app/mailers/user_mailer.rb

```diff
@@ -9,7 +9,7 @@ def confirmation_instructions(user, token, _opts = {})
     @token    = token
 
     I18n.with_locale(@resource.locale || I18n.default_locale) do
-      mail to: @resource.unconfirmed_email.blank? ? @resource.email : @resource.unconfirmed_email
+      mail to: @resource.unconfirmed_email.blank? ? @resource.email : @resource.unconfirmed_email, subject: I18n.t('devise.mailer.confirmation_instructions.subject')
```

Comment:
I would like to inject the title of the instance here. Any idea on how I would do that ?